bin/migrate-oats-data/prod_db_obfuscation/alcs/application/application.py

```diff
@@ -0,0 +1,293 @@
+                            {get_update_column_query("cove_farm_impact")}
+                            WHERE uuid = %s;
+                """
+
+                cursor.execute(
+                    sql_query,
+                    (
+                        fake.name(),
+                        fake_latin.sentence(nb_words=2),
+                        fake_latin.sentence(nb_words=2),
+                        fake_latin.sentence(nb_words=2),
+                        fake_latin.sentence(nb_words=2),
+                        fake_latin.sentence(nb_words=2),
+                        fake_latin.sentence(nb_words=2),
+                        fake_latin.sentence(nb_words=2),
+                        fake_latin.sentence(nb_words=2),
+                        fake_latin.sentence(nb_words=2),
+                        fake_latin.sentence(nb_words=2),
+                        fake_latin.sentence(nb_words=2),
+                        fake_latin.sentence(nb_words=2),
+                        fake_latin.sentence(nb_words=2),
+                        fake_latin.sentence(nb_words=2),
+                        fake_latin.sentence(nb_words=2),
+                        fake_latin.sentence(nb_words=2),
+                        fake_latin.sentence(nb_words=2),
+                        fake_latin.sentence(nb_words=2),
+                        fake_latin.sentence(nb_words=2),
+                        fake_latin.sentence(nb_words=2),
+                        fake_latin.sentence(nb_words=2),
+                        fake_latin.sentence(nb_words=2),
+                        fake_latin.sentence(nb_words=2),
+                        fake_latin.sentence(nb_words=2),
+                        r_id[0],
+                    ),
+                )
+
+            conn.commit()
+    except Exception as err:
+        conn.rollback()
+        logger.exception(err)
+
+
+@log_process(logger, "update_with_random_alcs_application_submission_review")
+@inject_conn_pool
+def _update_application_submission_review(conn=None):
+    fake = Faker()
+    fake_latin = Faker("la")
+
+    try:
+        with conn.cursor() as cursor:
+            cursor.execute("SELECT uuid FROM alcs.application_submission_review")
+            record_ids = cursor.fetchall()
+
+            for r_id in record_ids:
+                sql_query = f"""
+                            UPDATE alcs.application_submission_review 
+                            SET 
+                                {get_update_column_query("first_name")},
+                                {get_update_column_query("last_name")},
+                                {get_update_column_query("position")},
+                                {get_update_column_query("department")},
+                                {get_update_column_query("phone_number")},
+                                {get_update_column_query("email")},
+                                {get_update_column_query("ocp_bylaw_name")},
```

Comment:
is ocp necessary to obfuscate? Name of bylaw isn't exactly identifiable info

Reply:
The problem with old data is that sometimes people put info into the wrong field. This field is free input and that is the reason why I'm obfuscating it